doc/en/api/1.0.0/logging.yaml

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
Good, but many users will land on this one: https://docs.geoserver.org/latest/en/user/rest/api/index.html
Could you update it as well?

Reply:
done